courses/free-courses-hi.md

```diff
@@ -42,6 +47,13 @@
 * [Complete Android Development Course in Hindi](https://www.youtube.com/playlist?list=PLUhfM8afLE_Ok-0Lx2v9hfrmbxi3GgsX1) - Neat Roots
 
 
+### Bash and Shell
+
+* [shell scripting complete tutorial in hindi](https://youtube.com/playlist?list=PL9A0tISr5Ow5nZSY8-ICNAyXHJKesl_XL) - Cybersploit
+* [BASH SHELL SCRIPTING (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
```

Comment:
ALL CAPS titles not allowed.

Reply:
As suggested by Eric...

```suggestion
* [Bash Shell Scripting (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
```

Reply:
Fixed in commit 4b4b8b1bfd33a0075fb111b652e23feb42b60287